src/clr-angular/index.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+// export * from "./clr-angular.module";
+
+/**********
+ * Buttons
+ *
+ * Buttons are used to create click/touch handlers in a variety of use cases.
+ */
+export * from "./button/button.module";                     // causes build failure;
+export * from "./button/button-group/button-group.module";  // causes build failure;
+export * from "./button/button-group/index";                // causes build failure;
+export * from "./button/button-loading/loading-button.module";
+export * from "./button/button-loading/index";
+
+
+/**********
+ * Code
+ *
+ * code syntax highlighing is used for ????
+ */
+export * from "./code/index";
+export * from "./code/syntax-highlight/syntax-highlight.module";
+
+/*********
+ * Data
+ *
+ * There are three types of data component.
+ * - Datagrid holds rows of data organized with columns.
+ * - TreeView holds hierarchical data that needs to be traversed
+ * - StackView holds data with a  master / child relationshiop
+ */
+export * from "./data/data.module";                 // causes build failure;
+export * from "./data/datagrid/datagrid.module";    // causes build failure;
+export * from "./data/datagrid/index";              // causes build failure
+export * from "./data/tree-view/index";             // causes build failure
+export * from "./data/tree-view/tree-view.module";  // causes build failure
+export * from "./data/stack-view/index";
+export * from "./data/stack-view/stack-view.module";
+
+/*********
+ * Emphasis
+ *
+ * These components are used to draw attention to actionable information.
+ */
+export * from "./emphasis/alert/alert.module";  // causes build failure
+export * from "./emphasis/alert/index";
+export * from "./emphasis/emphasis.module";  // causes build failure
```

Comment:
Remove these build failure comments. Actually if you could revert this file to the original that would help me with rebasing.